tests/operators/test_activation.py

```diff
@@ -12,11 +12,16 @@
 import numpy as np
 import pytest
 import torch.nn.functional as F
+import torch.nn.functional as F
 
 from hidet import ops
 from hidet.testing import check_unary, check_binary, check_torch_binary, check_torch_unary
 
 
+# hidet operators tested against numpy equivalent operators
+from hidet.testing import check_unary, check_binary, check_torch_binary, check_torch_unary
+
+
```

Comment:
duplicated?

Reply:
It might help to start from the lastest main branch every time when we start a new work, should avoid many conflicts like this.